taccsite_cms/contrib/taccsite_sysmon/models.py

```diff
@@ -0,0 +1,45 @@
+from cms.models.pluginmodel import CMSPlugin
+from django.utils.translation import gettext_lazy as _
+
+from django.db import models
+
+from djangocms_attributes_field import fields
+
+from taccsite_cms.contrib.helpers import get_choices
+
+from .constants import DEFAULT_SYSTEM
+
+# TODO: (Maybe in GH-295) Do not replicate `display_name` data from API
+SYSTEM_DICT = {
```

Comment:
Is there a reason that this isn't in `constants.py`?

Reply:
I don't mind moving them (for this PR, and the other); `constants.py` is where I would naturally put them. Anyone else suggesting to do so is the only catalyst I need to do so.

Reply:
I refrain from moving constants, because:
1. [your "valid reason" comment/approval](https://github.com/TACC/Core-CMS/pull/297#issuecomment-897161469)
2. [my original reasoning](https://github.com/TACC/Core-CMS/pull/297#discussion_r687176885)
3. consistency with other recent plugins